wireguard-go-rs/build.rs

```diff
@@ -15,6 +15,8 @@ fn main() {
         "android" => {
             cmd.arg("--android");
         }
+        // TODO: Is is allowed to panic? Won't this always happen if we run `cargo build` in the
+        // workspace root for e.g. windows?
```

Comment:
ideally I think we shouldn't include wireguard-go-rs as a dependency on windows

Reply:
but if that's not possible then we should remove the panic, yes

Reply:
Don't think it's possible no. We don't have it as a dep for the daemon, but it's still compiled when just running `cargo build` in the workspace root. I removed the panic.